locales/base/translation.json

```diff
@@ -1419,8 +1419,9 @@
     },
     "accountNumber": {
       "label": "Account Number",
-      "placeholderText": "Enter your 10 digit account number",
-      "errorMessage": "Account number should be 10 digits long and only contain numbers"
```

Comment:
yeah you're right, it's safe to remove this because versions that need this will have it bundled anyway :)